DeskReserve/Data/DBContext/Entity/Room.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace DeskReserve.Data.DBContext.Entity
+{
+	[Table("Room")]
+	public class Room
+	{
+		[Key]
+		public Guid RoomId { get; set; }
+		public int RoomNumber { get; set; }
+		public int FloorId { get; set; }
```

Comment:
Is the FLoor Guid. Also you can fetch the desired relation entity with EntityFramework.